rllib/utils/serialization.py

```diff
@@ -113,8 +113,7 @@ def _discrete(sp: gym.spaces.Discrete) -> Dict:
     def _multi_binary(sp: gym.spaces.MultiBinary) -> Dict:
         return {
             "space": "multi-binary",
-            "n": int(sp.n),
-            "dtype": sp.dtype.str,
+            "n": sp.n,
```

Comment:
is sp.n an integer already, or is it an np.int. 

if it is an np.int it can't be serialized.

Reply:
integer or tuple of ints, actually